tests/Content/VersionTest.php

```diff
@@ -35,79 +106,326 @@ public function tearDown(): void
 	}
 
 	/**
-	 * @covers ::create
+	 * @covers ::content
+	 * @covers ::language
 	 */
-	public function testCreate(): void
+	public function testContentMultiLanguage(): void
 	{
+		$this->setUpMultiLanguage();
+
 		$version = new Version(
 			model: $this->model,
 			id: VersionId::published()
 		);
 
-		$this->assertFalse($version->exists());
+		Data::write($this->contentFile('en'), [
+			'title' => 'Test'
+		]);
 
-		$version->create([
+		Data::write($this->contentFile('de'), [
+			'title' => 'Töst'
+		]);
+
+		$this->assertSame('Test', $version->content('en')->get('title')->value());
+		$this->assertSame('Test', $version->content($this->app->language('en'))->get('title')->value());
```

Comment:
Added test for the default language:

```suggestion
		$this->assertSame('Test', $version->content($this->app->language('en'))->get('title')->value());
		$this->assertSame('Test', $version->content()->get('title')->value());
```

However I'm actually wondering if the default should actually be the current version for consistency with `$modelWithContent->content()`.

Reply:
We have some really nasty situations where we mix 'default' and 'current' as default language codes in model methods. I totally get why it feels like it should be 'current' here, but I also really cringe at the thought to mix 'current' and 'default' in multiple Version methods. 

I think we are having better chances to make a good decision when we arrive at the translation vs content nightmare.

Reply:
I agree. For now it's fine as it's internal anyway. And maybe we can write our own code in a way that is always explicit. I.e. the calling code never relies on the default. Then we'll have an easier time to refactor this later if needed.